docs/users/cli-usage.md

```diff
@@ -29,6 +29,109 @@ Available Commands:
 Use "ddev [command] --help" for more information about a command.
 ```
 
+## Quickstart
+ddev is designed to be as simple as possible to incorporate into existing Wordpress and Drupal workflows. You can start using ddev with any site just by running a few commands.
+
+Below are quickstart instructions for each app type; Wordpress, Drupal 7, and Drupal 8.
+
+**Note:** If you do not have ddev already on your machine, please follow the [installation instructions](https://ddev.readthedocs.io/en/latest/#installation) before beginning the quickstart tutorial. 
+### Wordpress
+To get started using ddev with a Wordpress site, simply clone the site's repository and checkout its directory.
+```
+git clone https://github.com/user/worpress_site
+cd wordpress_site
+```
+From here we can start setting up ddev. Inside of your site's working directory, enter the command:
+```
+ddev config
+```
+
+_Note: ddev config will prompt you for a site name and docroot._
+
+After you've run `ddev config`, you're ready to start running your site. To start running ddev, simply enter:
+```
+ddev start
+``` 
+When running `ddev start` you should see output informing you that the site's environment is being started. If startup is successful, you'll see a message like the one below telling you where the site can be reached.
+```
+Successfully started wordpress_site
+Your application can be reached at: http://wordpress_site.ddev.local
+```
+Quickstart instructions regarding database imports, can be found under [Database Imports](https://ddev.readthedocs.io/en/latest/users/cli-usage/#database-imports).
+### Drupal 7
+Beginning to use ddev with a Drupal 7 site is as simple as cloning the site's repository and checking out its directory.
+```
+git clone https://github.com/user/my_drupal7_site
+cd my_drupal7_site
+```
+Now to start working with ddev. Inside of your site's working directory, enter the following command:
+```
+ddev config
+```
+
+_Note: ddev config will prompt you for a site name and docroot._
+
+After you've run `ddev config` you're ready to start running your site. Run ddev using a simple:
+```
+ddev start
+``` 
+When running `ddev start` you should see output informing you that the site's environment is being started. If startup is successful, you'll see a message like the one below telling you where the site can be reached.
+```
+Successfully started my_drupal7_site
+Your application can be reached at: http://my_drupal7_site.ddev.local
+```
+Quickstart instructions regarding database imports, can be found under [Database Imports](https://ddev.readthedocs.io/en/latest/users/cli-usage/#database-imports).
+### Drupal 8
+Getting started with Drupal 8 sites in ddev is a friendly process that can be started by either cloning a git repository or using a new or existing composer project.
```

Comment:
Good usage on new or existing. Not sure I'd make big claims about friendly processes. How about "Start by getting the PHP code for your project by either cloning a git repository or using a new or existing composer project"